### PR TITLE
Chore/repo todo

### PR DESCRIPTION
**Pull request by maiali13**
# HRF Police Violence DS Team Pull Request Template
https://trello.com/c/hvYLjb7A

Repo cleaning:
- moved old twitter exploration files to ./notebooks/twitter
- deleted redundant csv files
- fix db setup seed scripts to refer to correct csv
- edit README for dbsetup folder
- added the new model pickle and renamed old .pkl file
- deleted old experimental and demo files

## Types of Changes

CHORE/repo_todo

## PR Creator Checklist

- [x] I wrote a label and description of the requested changes
- [x] PR was sent to multiple reviewers, including at least one team member and the TPL
- [x] I performed a self-review of my own code (followed the code review checklist)
- [x] I linked to the user story/Trello checklist item corresponding to the PR

## Code Review Checklist

Bearing in mind the [Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da):

- [ ] Atomic, well-named commits
- [ ] PR documents the motivation, user story and changes
- [ ] Code is consistently formatted 
- [ ] PR is appropriately scoped- small or only one issue
- [ ] Code is idiomatic and conforms to industry standards
- [ ] At least one team member or the TPL has reviewed the PR and left appropriate comments
- [ ] (When applicable) Reviewer suggestions were acted upon

### CODE REVIEW

**Review by axefx — APPROVED** [review]
Looks good, much needed clean up

File: dbsetup/README.md [hunk 1]
```diff
@@ -1,27 +1,5 @@
...
-check if incident id is greater than incident count
-else:
-create springfield record in places table
-
-add incident
-
-Check evidence
-
-- Query entire incidents table, and check evidence counts at each one
+This directory contains scripts used for the setup and seeding of the database using [psycopg2](https://www.psycopg.org/docs/). The database was initially populated with the incidents in the provided `training_data` csv. Tables are defined in the tables subfolder, seed scrips are found in the seed subfolder, and setup is completed when running `db_postgresql.py`.
```
Comment (michhottinger) [inline dbsetup/README.md:5]: nice description of what we used here.

File: dbsetup/clean_training-data.py [hunk 2]
```diff
@@ -50,4 +48,4 @@ def clean_pb2020(df):
 
 df = clean_pb2020(df)
 
-df.to_csv('dbsetup/training_data2.csv')
\ No newline at end of file
+df.to_csv('dbsetup/training_data.csv')
```
Comment (michhottinger) [inline dbsetup/clean_training-data.py]: good details

File: dbsetup/seed/evidences.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import csv
 import psycopg2
-from pprint import pprint
+
 
 def seed_table(conn):
     # evidence: id, link
-    with open('dbsetup/training_data2.csv', 'r') as f:
```
Comment (michhottinger) [inline dbsetup/seed/evidences.py:7]: thanks for updating this!

**Review by michhottinger — COMMENTED** [review]
Great updating and final touches. thanks for doing this.

**Pull request merged**